### PR TITLE
[etherorcs-combo-balanceof] Add EtherOrcs holder strategy

### PR DESCRIPTION
**Pull request by tempest-sol**
Fixes # . none

Changes proposed in this pull request:
- Add support for querying against the EtherOrcs graph to pull in owned counts since some assets are staked

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]

File: src/strategies/etherorcs-combo-balanceof/schema.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "properties": {
+        "address": {
+          "type": "string",
+          "title": "User address",
+          "examples": ["e.g. 0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984"],
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "minLength": 42,
+          "maxLength": 42
+        }
+      },
```
Comment (ChaituVR) [inline src/strategies/etherorcs-combo-balanceof/schema.json:17]: There is no address param right? 🤔

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/etherorcs-combo-balanceof/schema.json (hunk 1 above)
Comment (tempest-sol) [reply src/strategies/etherorcs-combo-balanceof/schema.json]: Correct, I made the adjustment.

File: src/strategies/etherorcs-combo-balanceof/schema.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Strategy",
+  "definitions": {
+    "Strategy": {
+      "title": "Strategy",
+      "type": "object",
+      "properties": { },
+      "required": ["address"],
```
Comment (ChaituVR) [inline src/strategies/etherorcs-combo-balanceof/schema.json:9]:
```suggestion
```

Comment (ChaituVR) [reply src/strategies/etherorcs-combo-balanceof/schema.json:9]: @tempest-sol Make sure to run the test cases in your local

Comment (tempest-sol) [reply src/strategies/etherorcs-combo-balanceof/schema.json]: I removed the schema since it's optional

**Comment by ChaituVR** [comment]
Thanks @tempest-sol

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Hey @tempest-sol I just noticed you didn't use `snapshot`, you should use this block number or its timestamp to make sure results are consistent. if it is a static API which returns same data everytime, this should be fine

**Pull request merged**